pyproject.toml

```diff
@@ -37,6 +37,7 @@ more-itertools = "^9.1.0"
 packaging = "^23.1"
 setuptools = "^68.0.0"
 gitpython = "^3.1.32"
+debugpy = "1.6.7"
```

Comment:
I think this should go under `dev.dependencies` but it probably doesn't matter much since we all just install everything. I don't really use `debugpy` in my personal environment; so, I don't know. Thoughs  @bruno-fs ?

Reply:
this should go under dev dependencies, unless running with debugger becomes an "user facing" feature